manim/mobject/geometry/polygram.py

```diff
@@ -169,18 +204,28 @@ def construct(self):
 
         for vertices in self.get_vertex_groups():
             arcs = []
-            for v1, v2, v3 in adjacent_n_tuples(vertices, 3):
+
+            # Repeat the radius list as necessary in order to provide a radius
+            # for each vertex.
+            if type(radius) is float:
+                radius_list = [radius] * len(vertices)
+            else:
+                radius_list = radius * ceil(len(vertices) / len(radius))
```

Comment:
What happens if I pass `radius=1`? Given that `type(1) is float` yields `False`, you rather might want to use `isinstance(radius, (int, float))` instead.

Reply:
Applied!